app/views/confirmation/account.html.erb

```diff
@@ -14,14 +14,11 @@
       What happens next?
     </h2>
     <p>
-      Reliability engineering will confirm your AWS account can be billed through the central account.
+      You will receive confirmation that your AWS account can be billed through the central account.
```

Comment:
Hm, we don't actually confirm accounts can be billed when new accounts are created. We probably want to say something like:

"Your request for a new AWS account will be reviewed, and if accepted, your new account will be created and the details will be emailed to you when ready."